src/System.Private.CoreLib/src/System/Diagnostics/Stackframe.cs

```diff
@@ -263,7 +267,17 @@ public override string ToString()
 
                     sb.Append('>');
                 }
+#if !CORECLR
+                includeFileInfoIfAvailable = true;
+            }
+            else
+            {
+                includeFileInfoIfAvailable = AppendStackFrameWithoutMethodBase(sb);
+            }
 
+            if (includeFileInfoIfAvailable)
+            {
+#endif
```

Comment:
@jkotas Please let me know if you prefer the way I included this #ifdef to be improved and whether or not you have suggestions for improving this part.